### DISCUSSION TITLE
2023/01/11/multiregion/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#152] [Blog Comments]
# 2023/01/11/multiregion/

안녕하세요. 이번 내용에서는 AWS 환경에서 멀티 리전을 통해 WordPress 웹 사이트를 재해복구 환경으로 구성하는 내용으로 작성하였습니다. 재해 복구(Disaster Re

https://tech.cloudmt.co.kr/2023/01/11/multiregion/

**Comment by shb003** [comment]
좋은 자료 정말 감사합니다! 그런데 혹시 ElasticBeanstalk 도쿄라인쪽은 어떻게 구성하신건지 여쭤봐도 괜찮을까요?
